### ISSUE TITLE
Theming not working as intended

### ISSUE THREAD

**Issue by sifrenette** [wix/react-native-calendars#4]
Hello, great component !

Changing the component's theme is not working as intended when using markingType="interactive". Days styles are not applied correctly (dayTextColor, todayTextColor, etc..). Here's two examples with markingType="interactive" and markingType="simple" and with only that prop changed.

[markingType="simple"](https://i.gyazo.com/cc101464dd9329db522b144b9b4f56c5.mp4)
[markingType="interactive"](https://i.gyazo.com/6fa7c9edc382ddbc3bc6a77660fd29fb.mp4)

Here's my code:
```
<CalendarList
                    markedDates={markedDates}
                    onDayPress={this.onPress}
                    markingType="interactive" // markingType="simple"
                    theme={{
                        calendarBackground: '#333248',
                        textSectionTitleColor: 'white',
                        dayTextColor: 'white',
                        todayTextColor: 'white',
                        selectedDayTextColor: 'white',
                        monthTextColor: 'white',
                        selectedDayBackgroundColor: '#333248'
                    }}
              />
```

I checked the source code a bit, and if I understand correctly, the days are rendered by a different component if markingType="interactive". I'm guessing this other component is handling the theme prop incorrectly?

**Comment by tautvilas** [comment]
Fixed in v1.2.7, added your code to calendar example :)

**Comment by sifrenette** [comment]
That was fast, thanks !
